assignment/scripts/aboutMe.js

```diff
@@ -1,44 +1,61 @@
 // REQUIRED FEATURES:
 // 1 - Create a variable called `firstName` and assign it the value of your first name
-
+let firstName= 'Brandon'
 // 2 - Create a second variable called `lastName` and assign it the value of your last name
-
+let lastName= 'Paulson'
 // 3 - Create a third variable called `fullName`, assign it the value of your first and last name
 // (remember, you already have variables for this, can you use those?)
-
+let fullName= (firstName+lastName)
```

Comment:
No need for parentheses here. When writing a JavaScript expression, they work a lot like they do in math, grouping operations and giving them priority in ordering so they get executed first.

```js
1 + 3 / 2;  // 2.5
```

```js
(1 + 3) / 2;  // 2
```

In this case, you only have the one operation so the parentheses is redundant.

Reply:
Also, the value of your `fullName` variable is going to be: `"BrandonPaulson"`! Don't forget the space.